doodad/mount.py

```diff
@@ -263,3 +263,25 @@ def dar_build_archive(self, deps_dir):
     def dar_extract_command(self):
         return 'echo helloMountAzure'
 
+
+class MountRemote(Mount):
```

Comment:
We might want to consider merging MountRemote, MountGCP, MountS3, and MountAzure into the same class later, but otherwise LGTM.

Reply:
Yeah, that's a good point. I think GCP, S3, and Azure could be merged together into something like "MountRemoteOutput" but I'll leave that for another PR.